src/index.js

```diff
@@ -15,6 +15,17 @@ module.exports = async function({
   if (!html) {
     throw Error('You must provide an html property.')
   }
+  let screeshotArgs = {};
+  if (/jpe?g/.test(type)) {
+    if (!quality) {
+      screeshotArgs.quality = 80;
+      quality = 80
+    } else {
+      screeshotArgs.quality = quality;
+    }
```

Comment:
A ternary could simplify this code a bit:

```suggestion
    screeshotArgs.quality = quality ? quality : 80
```

I also removed the line that set quality since it is not used afterwards.